proto/fungible/tx.proto

```diff
@@ -51,3 +52,11 @@ message MsgRemoveForeignCoin {
 }
 
 message MsgRemoveForeignCoinResponse {}
+
+message MsgUpdateContractBytecode {
+  string creator = 1;
+  string contract_address = 2;
+  bytes new_bytecode = 3;
```

Comment:
I think we should rather have an address pointing to an already deployed bytecode so it can be tested live before making the change, and potentially the upgrade process could be simpler. WDYT?